docs/root/api-v1/route_config/route.rst

```diff
@@ -378,6 +384,37 @@ regex
       "value": "POST"
     }
 
+.. _config_http_conn_man_route_table_route_query_args:
+
+Query Args
```

Comment:
Do you have a need for this in v1? If not, I'd suggest but not require that we make it v2 only.

Reply:
I only have a need for v1; we've built our internal control plane atop v1, and so far we haven't made any plans to invest in a v2 conversion.  From an Envoy core perspective, is it feasible to have features be v1-only, or is v2 support a necessity?

Reply:
v2 support is required (the core code uses v2 at this point so you can't avoid implementing it). It's fine to support v1 also assuming you are willing to add tests, etc.